crates/api_crud/src/user/create.rs

```diff
@@ -275,11 +277,27 @@ impl PerformCrud for Register {
       );
     } else {
       if email_verification {
+        let local_user_view = LocalUserView {
+          local_user: inserted_local_user,
+          person: inserted_person,
+          counts: PersonAggregates {
+            id: 0,
+            person_id: Default::default(),
+            post_count: 0,
+            post_score: 0,
+            comment_count: 0,
+            comment_score: 0,
+          },
+        };
+        // we check at the beginning of this method that email is set
+        let email = local_user_view
+          .local_user
+          .email
+          .clone()
+          .expect("email was provided");
         send_verification_email(
-          inserted_local_user.id,
-          // we check at the beginning of this method that email is set
-          &inserted_local_user.email.expect("email was provided"),
-          &inserted_person.name,
+          &local_user_view,
+          &email,
```

Comment:
Ah, iirc email is only in LocalUser, not the LocalUserSettings or Safe.

Reply:
Not sure what you mean, local_user_view contains LocalUser. And I dont think i changed any of the logic.